src/test/java/gov/acwi/wqp/etl/monitoringLocation/TransformMonitoringLocationIT.java

```diff
@@ -0,0 +1,102 @@
+
+import gov.acwi.wqp.etl.BiodataBaseFlowIT;
+import gov.acwi.wqp.etl.monitoringLocation.index.BuildMonitoringLocationIndexesFlowIT;
+import gov.acwi.wqp.etl.monitoringLocation.table.SetupMonitoringLocationSwapTableFlowIT;
+
+public class TransformMonitoringLocationIT extends BiodataBaseFlowIT {
+
+	@Autowired
+	@Qualifier("monitoringLocationFlow")
+	private Flow monitoringLocationFlow;
+	
+	public static final String EXPECTED_DATABASE_QUERY_INDEX = "select tablename, indexname, indexdef from pg_indexes where tablename='station_swap_biodata'";
+	public static final String EXPECTED_DATABASE_QUERY_TABLE = "select table_catalog, table_schema, table_name, table_type from information_schema.tables where table_name='station_swap_biodata'";
+
+	@PostConstruct
+	public void beforeClass() throws ScriptException, SQLException {
+		EncodedResource encodedResource = new EncodedResource(resource, Charset.forName("UTF-8"));
+		ScriptUtils.executeSqlScript(dataSource.getConnection(), encodedResource);
+	}
+
+	@Before
+	public void setup() {
+		testJob = jobBuilderFactory.get("monitoringLocationFlowTest")
+				.start(monitoringLocationFlow)
+				.build()
+				.build();
+		jobLauncherTestUtils.setJob(testJob);
+	}
+
+	@Test
+	@DatabaseSetup(		connection="wqp",		value="classpath:/testResult/wqp/station/empty.xml")
+	@DatabaseSetup(		connection="wqp",		value="classpath:/testData/nwis/station/nwisStation.xml")
+	@DatabaseSetup(		connection="biodata",	value="classpath:/testData/biodata/station/bioShareBiodataSite.xml")
+	@DatabaseSetup(		connection="biodata",	value="classpath:/testData/biodata/station/bioShareSample.xml")
+	@DatabaseSetup(		connection="biodata",	value="classpath:/testData/biodata/station/bioShareSampleType.xml")
+	@ExpectedDatabase(	connection="wqp",		value="classpath:/testResult/wqp/station/station.xml", assertionMode=DatabaseAssertionMode.NON_STRICT_UNORDERED)
+	public void transformMonitoringLocationStepTest() {
+		try {
+			JobExecution jobExecution = jobLauncherTestUtils.launchStep("transformMonitoringLocationStep", testJobParameters);
+			assertEquals(ExitStatus.COMPLETED, jobExecution.getExitStatus());
+		} catch (Exception e) {
+			e.printStackTrace();
+			fail(e.getLocalizedMessage());
+		}
+	}
+
+	@Test
+	@DatabaseSetup(		connection="wqp",		value="classpath:/testResult/wqp/station/empty.xml")
+	@DatabaseSetup(		connection="wqp",		value="classpath:/testData/nwis/station/nwisStation.xml")
+	@DatabaseSetup(		connection="biodata",	value="classpath:/testData/biodata/station/bioShareBiodataSite.xml")
+	@DatabaseSetup(		connection="biodata",	value="classpath:/testData/biodata/station/bioShareSample.xml")
+	@DatabaseSetup(		connection="biodata",	value="classpath:/testData/biodata/station/bioShareSampleType.xml")
+	@ExpectedDatabase( value="classpath:/testResult/wqp/monitoringLocation/indexes/all.xml",
+			assertionMode=DatabaseAssertionMode.NON_STRICT_UNORDERED,
+				table=BuildMonitoringLocationIndexesFlowIT.EXPECTED_DATABASE_TABLE,
```

Comment:
BuildMonitoringLocationIndexesFlowIT.EXPECTED_DATABASE_TABLE is now EXPECTED_DATABASE_TABLE_CHECK_INDEX

Reply:
Updated